packages/wp-now/src/run-cli.ts

```diff
@@ -71,6 +77,40 @@ export async function runCli() {
 				}
 			}
 		)
+		.command(
+			'php <filePath>',
+			'Run the php command passing the arguments for php cli',
+			(yargs) => {
+				commonParameters(yargs);
+				yargs.positional('filePath', {
+					describe: 'Path to the PHP file to run',
+					type: 'string',
+				});
+			},
+			async (argv) => {
+				const spinner = startSpinner('Running the php command...');
```

Comment:
Because the caller might expect to use the output for something else, I don't think `wp-now php` should provide any additional output.

Reply:
Removed the wp-now logs:
https://github.com/WordPress/wordpress-playground/pull/336/commits/5157e9e3a26c29a64c810c5393b092e571bbbfa5